src/WpfMath/DummyAtom.cs

```diff
@@ -46,27 +46,13 @@ public bool IsKern
             get { return this.Atom is SpaceAtom; }
         }
 
-        public void SetLigature(FixedCharAtom ligatureAtom)
-        {
-            this.Atom = ligatureAtom;
-            this.Type = TexAtomType.None;
-            this.IsTextSymbol = false;
-        }
-
         public CharFont GetCharFont(ITeXFont texFont)
         {
             return ((CharSymbol)this.Atom).GetCharFont(texFont);
         }
 
-        public override Box CreateBox(TexEnvironment environment)
-        {
-            if (this.IsTextSymbol)
-                ((CharSymbol)this.Atom).IsTextSymbol = true;
-            var resultBox = this.Atom.CreateBox(environment);
-            if (this.IsTextSymbol)
-                ((CharSymbol)this.Atom).IsTextSymbol = false;
-            return resultBox;
-        }

```

Comment:
This code looked unnecessary: it changes `IsTextSymbol` but I wasn't able to find a code path for somebody to observe that flag anywhere. The only place that flag is used is [`DummyAtom.cs:98`](https://github.com/ForNeVeR/wpf-math/blob/c748dff895334e8378b2845563241006aa7f7842/src/WpfMath/ScriptsAtom.cs#L98), and I couldn't imagine a situation when `CharSymbol.CreateBox()` could lead to `DummyAtom.CreateBox()`. So, it was an unnecessary mutation, and I just got rid of it.